src/qforte/abc/algorithm.py

```diff
@@ -53,22 +53,39 @@ class Algorithm(ABC):
     def __init__(self,
                  system,
                  reference=None,
-                 trial_state_type='reference',
+                 trial_state_type='occupation_list',
                  trotter_order=1,
                  trotter_number=1,
                  fast=True,
                  verbose=False,
                  print_summary_file=False):
 
         self._sys = system
-        if(reference==None):
+        self._trial_state_type = trial_state_type
```

Comment:
Is `trial_state` a standard name for a non-determinantal reference in quantum computing? I find the nomenclature confusing.

Reply:
This could probably be changed to '_state_prep_type', sometimes we use 'trial state' to describe the general vqe/pqe state in our article, but I will commit the variable name change.